README.md

```diff
@@ -7,34 +7,14 @@ This is an Android GUI for [WireGuard](https://www.wireguard.com/). It [opportun
 ## Building
 
 ```
-$ git clone --recurse-submodules https://git.zx2c4.com/wireguard-android
+$ git clone --recurse-submodules https://github.com/PearShadow/wireguard-android.git
 $ cd wireguard-android
 $ ./gradlew assembleRelease
 ```
 
 macOS users may need [flock(1)](https://github.com/discoteq/flock).
 
-## Embedding
+## Modifications
 
-The tunnel library is [on Maven Central](https://search.maven.org/artifact/com.wireguard.android/tunnel), alongside [extensive class library documentation](https://javadoc.io/doc/com.wireguard.android/tunnel).
-
-```
-implementation 'com.wireguard.android:tunnel:$wireguardTunnelVersion'
-```
-
-The library makes use of Java 8 features, so be sure to support those in your gradle configuration with [desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring):
-
-```
-compileOptions {
-    sourceCompatibility JavaVersion.VERSION_1_8
-    targetCompatibility JavaVersion.VERSION_1_8
-    coreLibraryDesugaringEnabled = true
-}
-dependencies {
-    coreLibraryDesugaring "com.android.tools:desugar_jdk_libs:1.1.5"
-}
-```
-
-## Translating
-
-Please help us translate the app into several languages on [our translation platform](https://crowdin.com/project/WireGuard).
+- Added application to load on boot and enter the TV activity. 
+- After successful connection on tunnel start intent to navigate to [com.example.nrg](https://github.com/PearShadow/nrg-android)
```

Comment:
Why did we remove a part of the original readme?  Also, please write this a bit more descriptively, not as two simple bullet points.